deploy-agent/requirements.txt

```diff
@@ -3,10 +3,9 @@
 # tests
 tox==1.6.1
 coverage
-pytest==2.3.5
+pytest
```

Comment:
it is better to pin down a version

Reply:
for testing only I would rather use the latest version, but happy to pin to a newer version of pytest if you want

Reply:
can you pin down to a latest version? i think it is better to have deterministic test

Reply:
I don't think it is actually, we don't pin coverage either

Reply:
i think we should always pin down to a version for pip. we are not using coverage, so it is ok to remove this dependency all together.